python-lib/dku_model_accessor/model_metadata.py

```diff
@@ -39,7 +39,7 @@ def _get_model_info_handler(saved_model_version_id):
     except Exception as e:
```

Comment:
Raise also the initial exception `e` and log it ?

Reply:
It is raised when we can not catch the exact issue.